src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.scss

```diff
@@ -0,0 +1,24 @@
+#passwordStrengthLevel {
```

Comment:
This file looks equal to ``src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.scss``, so IMO the classes should be relocated to ``frontend/src/styles.scss``.